### DISCUSSION TITLE
City Sample Integration

### DISCUSSION THREAD

**Discussion by Adriwin06** [Adriwin06/Ultimate-CommonUI-Menu-System#9] [General]
Discussion about the City Sample integration I'm working on (download link in the description of the latest Release). Share anything you want (bugs, screenshots, ideas, etc)

**Comment by kryokil** [comment]
just let you know City implementation of your menu is very cool

**Reply by Adriwin06** [reply]
Thanks you really much, it really encourage me to continue. I will update this integration to add more Gameplay elements, and add the latest features from my menu system soon.
